pvlib/iotools/pvgis.py

```diff
@@ -391,8 +391,8 @@ def read_pvgis_hourly(filename, pvgis_format=None, map_variables=True):
 
 
 def get_pvgis_tmy(latitude, longitude, outputformat='json', usehorizon=True,
-                  userhorizon=None, startyear=None, endyear=None, url=URL,
-                  map_variables=True, timeout=30):
+                  userhorizon=None, startyear=None, endyear=None,
+                  map_variables=True, url=URL, timeout=30):
```

Comment:
Unlike in `get_psm3`, this order is a change from 0.9.5, so I think it should be noted in the what's new.